llvm/test/CodeGen/DirectX/Metadata/resource-symbols.ll

```diff
@@ -11,22 +11,22 @@ define void @test() {
   ; Buffer<float4>
   %float4 = call target("dx.TypedBuffer", <4 x float>, 0, 0, 0)
       @llvm.dx.resource.handlefrombinding(i32 0, i32 0, i32 1, i32 0, i1 false, ptr @A.str)
-  ; CHECK: %TypedBuffer = type { <4 x float> }
+  ; CHECK: %"Buffer<float4>" = type { <4 x float> }
```

Comment:
Could you explain the `"` quotes around the types here? I've seen this before but I'm not the biggest fan of them stylistically. Is there something about the name that requires them? Guessing the `<`/`>` are the problem?

Reply:
The quotes are there exactly because the '<''>' characters.